src/components/ScholarshipForm.jsx

```diff
@@ -63,8 +61,30 @@ function ScholarshipForm({ scholarship }) {
     },
   });
 
-  let lintIssues = {};
-  if (activeStep === 1) lintIssues = lintReqs(formik.values);
+  const lintIssues = activeStep === 1 ? lintReqs(formik.values) : {};
+  function autoFill() {
+    const vals = formik.values.requirements;
+    const lintVals = lintIssues.reqs;
+    const updatedReqs = {};
+
+    const grades = [...(vals.grades || []), ...(lintVals.grades || [])];
+    const schools = [...(vals.schools || []), ...(lintVals.schools || [])];
+    const states = [...(vals.states || []), ...(lintVals.states || [])];
```

Comment:
Could maybe skip the length check below and do something like:
```suggestion
    const updatedReqs = {
      grades: lintVals.grades ? [...(vals.grades || []), ...(lintVals.grades || [])] : val.grades,
      schools: lintVals.schools ? [...(vals.schools || []), ...(lintVals.schools || [])] : val.schools,
      ...
```

I'm indifferent about this though as I'm not sure how `undefined` values would be treated in the else case.

Reply:
I'm going to keep it as is.